### ISSUE TITLE
The installation process is awful

### ISSUE THREAD

**Issue by GiantAcidLemon** [AILab-CVC/YOLO-World#496]
Thanks for sharing the work, but the installation process is awful! There are so many conflict between requirements.txt and your pyproject.toml. Could anyone tell what exactly the correct process to setup the environment?

**Comment by surinder-ram** [comment]
yes, same problem here. starting with cuda version (which do not work on new cards), to mim and to different pip problems.

it would have been much better to release a docker, with 15 Gbyte, than this....
